.travis.yml

```diff
@@ -1,21 +1,47 @@
 language: node_js
-node_js:
-- '8'
-- '6'
-- '4.5.0'
-os:
-- linux
-- osx
-addons:
-  apt:
-    sources:
-    - ubuntu-toolchain-r-test
-    packages:
-    - g++-5
-
+compiler: default
 matrix:
+  exclude:
+  # Disable the default build and use customized matrix only.
+  - compiler: default
   include:
+  # Node 4.5.0          Linux (Precise)     G++5.4.1
+  - os: linux
+    dist: precise
+    node_js: '4.5.0'
+    compiler: g++-5
+    addons:
+      apt:
+        sources:
+        - ubuntu-toolchain-r-test
+        - george-edison55-precise-backports
```

Comment:
'ubuntu-toolchain-r-test' is for g++5/g++6
'george-edison55-precise-backports' is for CMake 3.2+